charts/argo-cd/Chart.yaml

```diff
@@ -22,3 +22,4 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
     - "[Added]: Add server.service.sessionAffinity setting of the Service into account when deciding which backend Pod to use"
```

Comment:
Old comment needs removing as it's concatenating it upstream.

Reply:
Ok. Thanks. I will remove this other.